jerry-core/parser/js/js-parser-expr.c

```diff
@@ -621,8 +621,16 @@ parser_parse_class_body (parser_context_t *context_p, /**< context */
       {
         is_computed = true;
       }
-      else if (!is_static && parser_is_constructor_literal (context_p))
+      else if (is_static)
       {
+        if (lexer_compare_literal_to_string (context_p, "prototype", 9))
```

Comment:
Can we ensure that the current literal is ident or string?

Also, as I see we should also throw error in the following case :
```js
class A {
  static get prototyp\u{0065}() {}
}
```